compiler/rustc_target/src/spec/targets/aarch64_unknown_none_certified.rs

```diff
@@ -0,0 +1,12 @@
+use crate::spec::Target;
+
+pub(crate) fn target() -> Target {
+    let mut target = super::aarch64_unknown_none::target();
+
+    target.metadata.description = Some("Bare ARM64, hardfloat (certified)".into());
```

Comment:
Let's avoid a situation where this breaks if upstream changes names:

```suggestion
    target.metadata.description = target.metadata.description.map(|v| format!("{v} (certified)").into());
```